ui/roll20_character_sheet_0.2.html

```diff
@@ -169,10 +169,9 @@
 
     <!-- Rolling -->
     <div display="inline-block">
-      <button type="roll" value="!EPRoll"></button>
+      <button name="act_epRoll" class="actionRollButton epRoll" type="action"></button>
       &emsp;
-      <button type="roll" value="!EPRoll @{roll_modifier}">
-      </button>
+      <button name="act_epRollWithModifier" class="actionRollButton epRollWithModifier" type="action"></button>
```

Comment:
There are a few buttons affected by this issue. I think we'll need to convert them all.